src/product.html

```diff
@@ -0,0 +1,252 @@
+      <div>
+        <img src="../img/3.png" class="w-[66.44px] mb-[20px]" />
+        <p class="font-[700] text-[24px] leading-[140%] mt-[5px] mb-[5px]">Security and privacy</p>
+        <p class="font-[400] text-[16px] opacity-[0.87] mt-[10px]">Lorem ipsum dolor sit amet, consetetur sadipscing
+          elitr, sed diam nonumy.</p>
+      </div>
+      <div>
+        <img src="../img/3.png" class="w-[66.44px] mb-[20px]" />
+        <p class="font-[700] text-[24px] leading-[140%] mt-[5px] mb-[5px]">Security and privacy</p>
+        <p class="font-[400] text-[16px] opacity-[0.87] mt-[10px]">Lorem ipsum dolor sit amet, consetetur sadipscing
+          elitr, sed diam nonumy.</p>
+      </div>
+      <div>
+        <img src="../img/3.png" class="w-[66.44px] mb-[20px]" />
+        <p class="font-[700] text-[24px] leading-[140%] mt-[5px] mb-[5px]">Security and privacy</p>
+        <p class="font-[400] text-[16px] opacity-[0.87] mt-[10px]">Lorem ipsum dolor sit amet, consetetur sadipscing
+          elitr, sed diam nonumy.</p>
+      </div>
+      <div>
+        <img src="../img/3.png" class="w-[66.44px] mb-[20px]" />
+        <p class="font-[700] text-[24px] leading-[140%] mt-[5px] mb-[5px]">Security and privacy</p>
+        <p class="font-[400] text-[16px] opacity-[0.87] mt-[10px]">Lorem ipsum dolor sit amet, consetetur sadipscing
+          elitr, sed diam nonumy.</p>
+      </div>
+      <div>
+        <img src="../img/3.png" class="w-[66.44px] mb-[20px]" />
+        <p class="font-[700] text-[24px] leading-[140%] mt-[5px] mb-[5px]">Security and privacy</p>
+        <p class="font-[400] text-[16px] opacity-[0.87] mt-[10px]">Lorem ipsum dolor sit amet, consetetur sadipscing
+          elitr, sed diam nonumy.</p>
+      </div>
+      <div>
+        <img src="../img/3.png" class="w-[66.44px] mb-[20px]" />
+        <p class="font-[700] text-[24px] leading-[140%] mt-[5px] mb-[5px]">Security and privacy</p>
+        <p class="font-[400] text-[16px] opacity-[0.87] mt-[10px]">Lorem ipsum dolor sit amet, consetetur sadipscing
+          elitr, sed diam nonumy.</p>
+      </div>
+    </div>
+  </section>
+  <section class="mr-[5%] ml-[5%]  mt-[120px] gap-[40px] grid grid-cols-2 mb-[100px]">
+    <div class="m-auto">
+      <div class="hero__bg">
+      </div>
+      <p
+        class="text-[24px] font-semibold bg-clip-text text-transparent bg-gradient-to-r from-[#373FFF] to-[#3ACAF8] mb-[20px] ">
+        Why should you work with us?</p>
+      <p class="text-[56px] font-[600] leading-[110%] mb-[20px] text-[#1D2130] ">To upscale your business to the next
+        level</p>
+      <p class="text-[16px] leading-[150%] font-[400]">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+        nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua invidunt ut labore.</p>
+    </div>
+    <div class="m-auto info__item--right">
```

Comment:
modifier with no preceeding element class. There should also be a block to contain the element.